test/ui-e2e/run-ui-tests.sh

```
@@ -1,16 +1,28 @@
+while [[ "$#" -gt 0 ]]; do
+    case $1 in
+        --env=*) ENV="${1#*=}" ;;
+        *) TEST_ARGS+=("$1") ;; # Save all other args (files, --headed, etc.)
+    esac
+    shift
+done
+
 if [ -f .env ]; then
   echo "Loading variables from .env file..."
   set -a  #export all variables
   source .env
```

Shellcheck (0.11.0)

[info] Not following: .env was not specified as input (see shellcheck -x).

(SC1091)